components/counts.py

```diff
@@ -18,19 +18,19 @@ def compute_counts():
         counts_for += [300, 400, 500, 600, 700, 800, 900, 1000, 1500, 2000]
         limit = 2100
 
-    champ_results = TourneyResult.objects.filter(league=champ, public=True).order_by("-date")
+    legend_results = TourneyResult.objects.filter(league=legend, public=True).order_by("-date")
```

Comment:
I think people often consult old counts, at least I've seen people posting old screenshots from this page, so likely will introduce a selector to choose the league and draw the counts for that league.